internal/pkg/coordinator/monitor.go

```diff
@@ -132,15 +133,15 @@ func (m *monitorT) Run(ctx context.Context) (err error) {
 		case hits := <-s.Output():
 			err = m.handlePolicies(ctx, hits)
 			if err != nil {
-				return err
+				m.log.Info().Err(err).Msgf("Encountered an error while policy leadership changes; continuing to retry.")
```

Comment:
i think these should be at least warn?

Reply:
i was thinking the same, but i followed the [levels in the index monitor](https://github.com/elastic/fleet-server/blob/master/internal/pkg/monitor/monitor.go#L224-L236), though at this point i think these log lines are providing a different function. two independent notes seems like enough to up the log level.